nemo/collections/nlp/models/language_modeling/megatron_bert_model.py

```diff
@@ -212,7 +327,9 @@ def process_batch(self, batch):
         datatype = torch.int64
 
         data = batch
-        data_b = tensor_parallel.broadcast_data(keys, data, datatype)
+        data_b = tensor_parallel.broadcast_data(
+            keys, data, datatype
+        )  # IS THIS NEEDED (NOT PRESENT IN GPT3 BUT PRESENT IN BERT NO-BATCH IMPLEMENTATION)
```

Comment:
It is not needed. data is distributed for all the workers correctly. no need of broadcast.